apps/astarte_appengine_api/priv/static/astarte_appengine_api.yaml

```diff
@@ -671,6 +673,8 @@ paths:
                       detail: Endpoint not found
         '405':
           description: Invalid object
+        '422':
+          description: Tried to unset a property with `allow_unset` false
```

Comment:
If I am not mistaken, the POST method is meant for publishing datastream values.
So this code path shouldn't involve (un)setting properties, and the 422 error here is not needed.